docs/development/README.md

```diff
@@ -156,3 +157,15 @@ Pocket
 ├── utility          # Implementation of the Utility module
 ├── Makefile         # [to-be-deleted] The source of targets used to develop, build and test
 ```
+
+### Linters
```

Comment:
Please add a section explaining "how does one add new linters?"

Reply:
@Olshansk I've added more documentation, including the part on how to manage linters, and how to write in-house checks/rules.

Reply:
LGTM!